integration/node/cmd_tx_bank_send_test.go

```diff
@@ -19,6 +20,12 @@ import (
 	envtest "github.com/ignite/cli/integration"
 )
 
+func waitForNextBlock(t *testing.T, client cosmosclient.Client) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
```

Comment:
Good idea, I think using `env.Ctx()` in favor of `t.Deadline()` is fine because the underlying context is cancelled when the test finishes (so when the deadline is reached).

done here 92bfebb4fc49d7923f779b8f09831824cd227b8c